plover/oslayer/log_osx.py

```diff
@@ -19,7 +19,6 @@ def emit(self, record):
         # Notification Center has no levels or timeouts.
         notification = NSUserNotification.alloc().init()
 
-        notification.setTitle_(__software_name__.capitalize())
         notification.setSubtitle_(record.levelname.title())
```

Comment:
Can we make this the notification title instead?

Reply:
I'll try it out

Reply:
Should this be kept for older macOS versions?

Reply:
Looks like we were the odd ones out -- on older versions of Mac, they didn't include the app name, just the icon:

![image](https://user-images.githubusercontent.com/5840970/115163574-d335c180-a077-11eb-823f-9a27958a8a4a.png)

and this PR on big sur:

![image](https://user-images.githubusercontent.com/5840970/115163596-ec3e7280-a077-11eb-967b-9f783c4832d5.png)